src/main/java/teammates/ui/controller/InstructorFeedbackResultsPageAction.java

```diff
@@ -106,70 +97,68 @@ protected ActionResult execute() throws EntityDoesNotExistException {
         data.sections = logic.getSectionNamesForCourse(courseId);
         String questionNumStr = getRequestParamValue(Const.ParamsNames.FEEDBACK_QUESTION_NUMBER);
         if (data.selectedSection.equals(ALL_SECTION_OPTION) && questionNumStr == null) {
-            data.bundle = logic.getFeedbackSessionResultsForInstructorWithinRangeFromView(
-                    feedbackSessionName, courseId, data.instructor.email, queryRange, data.sortType);
+            data.bundle = logic
+                    .getFeedbackSessionResultsForInstructorWithinRangeFromView(feedbackSessionName, courseId,
+                                                                               data.instructor.email,
+                                                                               queryRange, data.sortType);
         } else if (data.sortType.equals("question")) {
-            if(questionNumStr == null){
-                data.bundle = logic.getFeedbackSessionResultsForInstructorInSection(
-                                feedbackSessionName, courseId,
-                                data.instructor.email, data.selectedSection);
+            if (questionNumStr == null) {
+                data.bundle = logic.getFeedbackSessionResultsForInstructorInSection(feedbackSessionName, courseId,
+                                                                                    data.instructor.email,
+                                                                                    data.selectedSection);
```

Comment:
This is OK too (although it breaks the line wrapping rules, the readability remains similar while we save a line).
```
data.bundle = logic.getFeedbackSessionResultsForInstructorInSection(feedbackSessionName, courseId,
                                                          data.instructor.email,  data.selectedSection);
```